src/sagas.js

```diff
@@ -697,13 +697,19 @@ function adjustVVFile ({ data, options, usbFilter }) {
   if (usbFilter) {
     data = data.replace(/^\[virt-viewer\]$/mg, `[virt-viewer]\nusb-filter=${usbFilter}`)
   }
+  if (options && isSpice) {
+    const smartcardEnabled = options.get ? options.get('smartcardEnabled') : options.smartcardEnabled
+    data = data.replace(/^enable-smartcard=[01]$/mg, `enable-smartcard=${smartcardEnabled ? 1 : 0}`)
```

Comment:
Why to we need `m` regexp flag when the regexp contains both `^` and `$` and no linebreaks?

Reply:
`data` is multi-line string, thus we need this modifier.